lib/storage-delegations.js

```diff
@@ -0,0 +1,193 @@
+/*!
+ * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const assert = require('assert-plus');
+const bedrock = require('bedrock');
+const database = require('bedrock-mongodb');
+const {promisify} = require('util');
+const {util: {BedrockError}} = bedrock;
+
+bedrock.events.on('bedrock-mongodb.ready', async () => {
+  await promisify(database.openCollections)(['zcap-delegation']);
+
+  // FIXME: what are indexes for this collection?
+
+  await promisify(database.createIndexes)([{
+    collection: 'zcap-delegation',
+    fields: {invocationTarget: 1, id: 1},
```

Comment:
I wonder if we should add `id` on its own as a unique index as well. This is different from other cases where we're storing authorized or revoked zcaps for other parties. This storage is for zcaps that have been locally generated and delegated to other parties, so that system should have control over the `id` field and we would want, I would think, to reject duplicate IDs. Thoughts?

Reply:
I believe the other unique index on controller (possilby delegator) and id is adequate.

Reply:
Delgations have been removed.